lib/hanami/cli/commands/db/rollback.rb

```diff
@@ -0,0 +1,39 @@
+module Hanami
+  class CLI
+    module Commands
+      module Db
+        # @since x.x.x
+        # @api private
+        class Rollback < Command
+          requires "model.sql"
+
+          desc "Rollback the database"
+
+          argument :versions, desc: "Number of versions to rollback the database", default: 1
+
+          example [
+            "  # Rollback only one version (default)",
```

Comment:
WDYT about: `Rollback only latest migration version` or something like this?

Reply:
`Rollback lastest version`?